raven/processes/wps_graph_ensemble_uncertainty.py

```diff
@@ -0,0 +1,69 @@
+from pywps import ComplexInput, ComplexOutput
+from pywps import Format
+from pywps import Process
+import zipfile
+from pathlib import Path
+from raven.utilities.graphs import mean_annual_hydrograph, hydrograph
+from matplotlib import pyplot as plt
+
+class GraphEnsUncertaintyProcess(Process):
+    def __init__(self):
+        inputs = [ComplexInput('sims', 'Stream flow simulations ensemble',
+                               abstract='Stream flow simulation time series',
+                               supported_formats=(Format(mime_type='application/zip'),)),
```

Comment:
Add the netCDF Format as well, so people can submit a list of files.

Reply:
also, you can use 
```
from pywps import FORMATS
supported_formats=(FORMATS.NETCDF)
```

Reply:
Done